include/powsybl/iidm/Identifiable.hpp

```diff
@@ -54,12 +59,16 @@ class Identifiable : public virtual Validable, public Extendable, public MultiVa
 
     stdcxx::const_range<std::string> getPropertyNames() const;
 
+    bool hasAliases() const;
```

Comment:
I backported this method, but unlike in Java, this method is not really useful in C++ : 
 - in Java, it is needed to know whether a **start** or an **empty** element should be written to generate an optimized XML (i.e to generate `<element />` instead of `<element></element>`)
 - in C++, the libXml2 API is quite different and both start+end elements have to be written, eventhe element have no child (i.e to generate an "empty" element in Java semantic)

So, should I remove it ?